src/validations/rules/ssp.sch

```diff
@@ -137,8 +147,59 @@
         <sch:active
             pattern="user-properties" />
     </sch:phase>
+
+    <sch:phase
+        id="remote-resource">
+        <sch:active
+            pattern="fedramp-data" />
+        <sch:active
+            pattern="fips-140" />
+    </sch:phase>
+
     <doc:xspec
         href="../test/ssp.xspec" />
+
+    <!-- settings for use (or not) of remote resources -->
+    <!-- XSLT parameter USE_REMOTE_RESOURCES -->
+    <xsl:param
+        as="xs:boolean"
+        name="USE_REMOTE_RESOURCES"
```

Comment:
Minor, but the other `xsl:params` use lower dash case, eg `registry-base-path`. I don't have a preference, but think we should switch this other the others to be consistent with each other.